tests/MongoDB.Driver.Tests/AggregateGraphLookupEnumerableFromOrToTests.cs

```diff
@@ -0,0 +1,241 @@
+﻿/* Copyright 2020-present MongoDB Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System.Collections.Generic;
+using FluentAssertions;
+using MongoDB.Bson;
+using MongoDB.Driver.Core.Misc;
+using MongoDB.Driver.Core.TestHelpers.XunitExtensions;
+using Xunit;
+
+namespace MongoDB.Driver.Tests
+{
+    public class AggregateGraphLookupTests
```

Comment:
I enjoyed reading these tests as I found them to provide clear examples of a feature I was unfamiliar with prior to starting this PR :) 😎🎶

Reply:
I wonder if we should link to this test in our docs here https://mongodb.github.io/mongo-csharp-driver/2.11/reference/driver/crud/reading/#aggregation to provide an example of how `Graphlookup` works?

A heading titled "GraphLookup" under and at the same level of `StageOperators` that links to this test class and https://docs.mongodb.com/manual/reference/operator/aggregation/graphLookup/ perhaps?

e.g.

[`$GraphLookup`](https://docs.mongodb.com/manual/reference/operator/aggregation/graphLookup/) is supported via the `GraphLookup` method and its overloads. See this [test class](https://github.com/mongodb/mongo-csharp-driver/blob/master/tests/MongoDB.Driver.Tests/AggregateGraphLookupEnumerableFromOrToTests.cs) for examples of usage.

Reply:
Change the class name also to match the file name.

Reply:
While I have often said that we should write our tests assuming that our users might look at the tests as an example of how to use a feature, I don't think our documentation should point to the tests (even if we might have done so now and then in the past).

Reply:
> Change the class name also to match the file name.

Done.

I didn't change the documentation because of Robert's concerns.